mpd/client.py

```diff
@@ -103,8 +104,60 @@ def do_connect(self):
 else:
     _BLISSIFY_DATA_HOME = os.path.expanduser("~/.local/share/blissify")
 
+# Distance between two songs
```

Comment:
Better to use [docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).